src/components/common/Button/Button.jsx

```diff
@@ -0,0 +1,9 @@
+const Button = ({ className, onClick, children }) => {
+  return (
+    <button className={`${className}`} onClick={onClick}>
```

Comment:
요렇게 해도 동일할 것 같은데 템플릿 리터럴을 사용해주신 이유가 있으실까요?
```suggestion
    <button className={className} onClick={onClick}>
```

Reply:
Button 컴포넌트를 작성할때 "분명 여러 곳에서 쓰면서 조건에 따라 클래스 이름을 동적으로 바꿔야겟지?" 라는 생각으로 템플릿 리터럴을 사용한것 같습니다.
하지만 아직 그런 부분들이 없지만 후에 분명 추가가 될 것 같아서 수정하지 않았었습니다!